subcell_pipeline/analysis/compression_metrics/polymer_trace.py

```diff
@@ -61,7 +61,7 @@ def get_end_to_end_axis_distances_and_projections(
 def get_average_distance_from_end_to_end_axis(
     polymer_trace: np.ndarray,
     **options: Dict[str, Any],
-) -> float:
+) -> np.float_:
```

Comment:
Is there a reason this is `np.float_` instead of `float`?

Reply:
Ah yes that was the issue with some of the other ones. You can call `.item()` on most numpy values to get the native type back (so `np.nanmean(perp_distances).item()`). Casting to a float works too!